cmd/waku/server/rest/health.go

```diff
@@ -0,0 +1,48 @@
+package rest
+
+import (
+	"context"
+	"errors"
+	"net/http"
+
+	"github.com/go-chi/chi/v5"
+	"github.com/waku-org/go-waku/waku/v2/node"
+)
+
+type HealthService struct {
+	node *node.WakuNode
+	mux  *chi.Mux
+}
+
+const routeHealth = "/health"
+
+func NewHealthService(node *node.WakuNode, m *chi.Mux) *HealthService {
+	h := &HealthService{
+		node: node,
+		mux:  m,
+	}
+
+	m.Get(routeHealth, h.getHealth)
+
+	return h
+}
+
+type HealthResponse string
+
+func (d *HealthService) getHealth(w http.ResponseWriter, r *http.Request) {
+	isReady, err := d.node.RLNRelay().IsReady(r.Context())
+	if err != nil {
+		if errors.Is(err, context.DeadlineExceeded) {
+			writeResponse(w, HealthResponse("Health check timed out"), http.StatusInternalServerError)
+		} else {
+			writeResponse(w, HealthResponse(err.Error()), http.StatusInternalServerError)
+		}
+		return
+	}
+
+	if isReady {
+		writeResponse(w, HealthResponse("Node is healthy"), http.StatusOK)
```

Comment:
Now i see this, i am doubting if this should be health or readiness check? Maybe better to have 2 different endpoints.

I like the pattern [k8s](https://medium.com/@AADota/kubernetes-liveness-and-readiness-probes-difference-1b659c369e17) follows for services . It has liveness and readiness checks which are separate.